specificatie/nationaliteit.yaml

```diff
@@ -0,0 +1,80 @@
+openapi: 3.0.3
+info:
+  title: Nationaliteit definities
+  version: 2.0.0
+  contact: {}
+paths: {}
+components:
+  schemas:
+    AbstractNationaliteit:
+      type: object
+      required:
+        - type
+      properties:
+        type:
+          type: string
+        redenOpname:
+          $ref: 'https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.2.0/api-specificatie/common.yaml#/components/schemas/Waardetabel'
+      discriminator:
+        propertyName: type
+        mapping:
+          GbaNationaliteit: '#/components/schemas/GbaNationaliteit'
+          Nationaliteit: '#/components/schemas/Nationaliteit'
+          BehandeldAlsNederlander: '#/components/schemas/BehandeldAlsNederlander'
+          VastgesteldNietNederlander: '#/components/schemas/VastgesteldNietNederlander'
+    Nationaliteit:
+      description: |
+        * **redenOpname** : De reden op grond waarvan de persoon de nationaliteit gekregen heeft.
+      allOf:
+        - $ref: '#/components/schemas/AbstractNationaliteit'
+        - type: object
+          properties:
+            datumIngangGeldigheid:
+              $ref: 'datum.yaml#/components/schemas/AbstractDatum'
+            nationaliteit:
+              $ref: 'https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.2.0/api-specificatie/common.yaml#/components/schemas/Waardetabel'
+            inOnderzoek:
+              $ref: '#/components/schemas/NationaliteitInOnderzoek'
+    BehandeldAlsNederlander:
+      allOf:
+        - $ref: '#/components/schemas/AbstractNationaliteit'
+        - type: object
+          properties:
```

Comment:
Hierin, en in VastgesteldNietNederlander ontbreekt inOnderzoek. Ik zou hier wel inOnderzoek verwachten, met aanduidingBijzonderNederlanderschap

Reply:
toegevoegd